src/hb-set.cc

```diff
@@ -61,6 +61,41 @@ hb_set_create ()
   return set;
 }
 
+/**
+ * Creates a set initially populated with num_codepoint codepoints, which must
+ * be in increasing order.
+ *
+ * Since: REPLACEME
+ */
+HB_EXTERN void
+hb_set_add_sorted_array (hb_set_t             *set,
+		         const hb_codepoint_t *sorted_codepoints,
+		         unsigned int          num_codepoints)
+{
+  if (unlikely(!set || !sorted_codepoints || !num_codepoints)) return;
```

Comment:
We don't support `nullptr` in our functions. The rest need to be pushed down inside `add_sorted_array`.